internal/services/monitor/monitor_activity_log_alert_resource.go

```diff
@@ -57,6 +59,21 @@ func resourceMonitorActivityLogAlert() *pluginsdk.Resource {
 
 			"resource_group_name": commonschema.ResourceGroupName(),
 
+			"location": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Default:  "global",
+				ValidateFunc: validation.Any(
+					location.EnhancedValidate,
+					validation.StringInSlice([]string{
+						"global",
+					}, false),
+				),
+				StateFunc:        location.StateFunc,
+				DiffSuppressFunc: location.DiffSuppressFunc,
+			},
```

Comment:
I think this could be replaced with 
```suggestion
			"location": commonschema.LocationOptional(),
```

Reply:
you're right. I checked and the default location value is still "global", so this wouldn't be a breaking change for existing resources. I made the change.